setup.py

```diff
@@ -2,13 +2,13 @@
 
 from setuptools import setup, find_packages
 
-__version__ = '0.4.3.0.1'
+__version__ = '1.4.3.0.1.1'
 
 setup(name="adminapi",
       version=__version__,
       description="Eucalyptus Cloud Services and General System Administrative Utilities",
       long_description="Eucalyptus Cloud Services and General System Administrative Utilities",
-      url="https://github.com/bigschwan/adminapi",
+      url="https://github.com/nephomaniac/adminapi",
```

Comment:
Just curious, why these and not https://github.com/eucalyptus/nephoria ? (I'd read this PR to understand more about domains/regions and AWS vs Euca)